src/modules/layout/components/Toolbar.js

```diff
@@ -24,22 +26,33 @@ const ToolbarContainer = styled.div`
 
   @media ${props => props.theme.dimensions.smallViewport} {
     width: 100%;
-    flex-direction: row;
+    flex-flow: row wrap;
     justify-content: center;
   }
 `
 
+const Headline = styled.h5`
+  width: 100vw;
+  margin: 0;
+  text-align: center;
+  font-size: 90%;
+`
+
 type PropsType = {|
   className?: string,
-  children?: Node
+  children?: Node,
+  viewportSmall: boolean,
+  t: TFunction
 |}
 
 class Toolbar extends React.PureComponent<PropsType> {
   render () {
-    return <ToolbarContainer className={this.props.className}>
+    const { viewportSmall, t } = this.props
+    return <ToolbarContainer className={this.props.className} viewportSmall={viewportSmall}>
+      {viewportSmall && <Headline>{t('isThisSiteUseful')}</Headline>}
       {this.props.children}
     </ToolbarContainer>
   }
 }
 
-export default Toolbar
+export default withTranslation('feedback')(Toolbar)
```

Comment:
Would it be possible to move that up as well to LocationLayout or any other parent component?

Reply:
Could be, but not the slim way imo. The reason to place it here was, that it's the generic toolbar container component that imports all necessary items at least.

Reply:
Yep, okay for me :+1: